pkg/workload/schemachange/error_screening.go

```diff
@@ -98,6 +98,18 @@ func (og *operationGenerator) schemaExists(
 	)`, schemaName)
 }
 
+func (og *operationGenerator) fnExists(
+	ctx context.Context, tx pgx.Tx, fnName string,
+) (bool, error) {
+	// TODO(before merge): we want to use specfic_name
```

Comment:
@rafiss I am still looking at this, but just in case you know off of the top of your head: is there a way we can check if the newly created fn is a duplicate by name and param types of any other existing fn?

using specific_name sounds nice, but i am not too sure on how we get from the template SQL string to its specific_name

Reply:
this could be a good starting point:
```
select proname, proargnames from pg_proc where proname = 'f';
```

(technically that just matches on the names. to check types also, it would be in `proargtypes`)

Reply:
gotcha - it just looks like the set of param types matters here; so, i just focused on that! thank you